src/main/java/de/tsystems/mms/apm/performancesignature/util/PerfSigUtils.java

```diff
@@ -145,13 +144,8 @@ public static Area calcDefaultSize() {
     }
 
     public static UsernamePasswordCredentials getCredentials(final String credsId) {
```

Comment:
OK since you clarified that this is a global configuration. You would need to pass the item if this setting was configurable per job or per folder (having a hierarchical configuration with overrides per-folder is nice for users, but not part of the Credentials API migration that I am asking for)